src/foam/nanos/auth/UpdatePassword.js

```diff
@@ -165,7 +165,7 @@ foam.CLASS({
       code: function() {
         this.auth.updatePassword(null, this.originalPassword, this.newPassword)
         .then((result) => {
-          this.user.copyFrom(result);
+          this.subject.user.copyFrom(result);
```

Comment:
if the auth.updatePassword() only changes the user password, might not need to copy back to the subject.user on the client side as the password field is networkTransient.